handbook/rbac/rbac.md

```diff
@@ -120,15 +120,15 @@ User's are backed by an identity provider (IdP) such as LDAP, Active Directory,
 If a user is  a member of multiple workspaces, the resources in those workspaces will be shown simultaneously within the grids and other views within SystemLink. This is useful when users need to view a rollup of resources across multiple workspaces. While resources in multiple workspaces may be viewed in a single grid, the actions a user can take against those resources may be different depending on their role in each workspace.
 
 !!! note "Considerations for WebVIs, dashboards, and Jupyter Notebooks"
-    The privileges for WebVIs and dashboards control the create, view, update, and delete privileges for the WebVI and dashboard documents themselves. When a users opens and views a dashboard or WebVI the data they can access is determined by the privileges on the resources exposed by the WebVI or dashboard such as tags, queries, and notebooks. If a user does not have privileges to access the resources exposed by the WebVI or dashboard they will view no data.
+    The privileges for WebVIs and dashboards control the create, view, update, and delete privileges for the WebVI and dashboard documents themselves. When a user opens and views a dashboard or WebVI the data they can access is determined by the privileges on the resources exposed by the WebVI or dashboard such as tags, queries, and notebooks. If a user does not have privileges to access the resources exposed by the WebVI or dashboard they will view no data.
 
-    When a user runs a Jupyter Notebook from a *Report* or dashboard, the Notebook executes with the same privileges as the user. This allows the creation of Notebooks without workspace logic. For example, the Systems Management dashboard includes a tile displaying the number of alarms for various systems. This tile will only show systems within the logged in user's workspaces. If the user does not have privileges to view alarms, the tile returns no data.
+    When a user runs a Jupyter Notebook from a *Report* or dashboard, the Notebook executes with the same privileges as the user. This allows the creation of Notebooks without workspace logic. For example, the Test Insights dashboard includes a tile displaying system utilization over time. This tile will only show utilization for systems within the logged in user's workspaces. If the user does not have privileges to view systems, the tile returns no data.
```

Comment:
I changed this example to use the Test Insights dashboard because the Systems Management dashboard has been replaced. Let me know if this example is not helpful.